esphome/components/api/api_connection.cpp

```diff
@@ -698,6 +698,49 @@ void APIConnection::button_command(const ButtonCommandRequest &msg) {
 }
 #endif
 
+#ifdef USE_LOCK
+bool APIConnection::send_lock_state(lock::Lock *a_lock, lock::LockState state) {
+  if (!this->state_subscription_)
+    return false;
+
+  LockStateResponse resp{};
+  resp.key = a_lock->get_object_id_hash();
+  resp.state = (enums::LockState) state;
```

Comment:
To be consistent with other code in here
```suggestion
  resp.state = static_cast<enums::LockState>(state);
```